### PR TITLE
css form login-register-order-contact--responsite

### PR DESCRIPTION
**Pull request by comadaihiep92**

### CODE REVIEW

**Comment by comadaihiep92** [comment]
ready

File: src/pug/utils/mixins.pug [hunk 1]
```diff
@@ -296,7 +298,11 @@ mixin orderButton(list)
 mixin blog2(list)
     - var listBlog= list
     - var count = 0
+<<<<<<< HEAD
     - var n = 3
+=======
+    - var n = 'all'
+>>>>>>> css form login-register-order-contact--responsite
```
Comment (chienpv-0644) [inline src/pug/utils/mixins.pug]: gì đây em (tat)

**Comment by chienpv-0644** [comment]
commented

**Comment by comadaihiep92** [comment]
ready

File: dist/pages/blog-page.html [hunk 2]
```diff
@@ -54,6 +60,105 @@ <h1 class="blog__title">Vang thang long classic</h1>
           <p class="blog__description">Vang nổ Thăng Long có hương vị đặc trưng của sản phẩm lên men tự nhiên từ hoa quả với độ rượu nhẹ, bọt ga đầy trắng mịn. Vang Nổ thăng long tạo cảm giác hương phấn, êm dịu, vui tươi, sản phẩm được đóng chai dung tích 750ml...</p><a class="blog__link" href="#">Read more</a>
         </div>
       </div>
+<<<<<<< HEAD
+=======
```
Comment (chienpv-0644) [inline dist/pages/blog-page.html]: (2tat)

File: dist/pages/blog-page.html [hunk 3]
```diff
@@ -54,6 +60,105 @@ <h1 class="blog__title">Vang thang long classic</h1>
...
+          <h1 class="blog__title">Vang thang long classic</h1>
+          <div class="blog__info">
+            <p class="blog__author">Đăng bởi Giangle</p>
+            <p class="blog__date">30/06/2015</p>
+            <p class="blog__comment">60 bình luận</p>
+          </div>
+          <p class="blog__description">Vang nổ Thăng Long có hương vị đặc trưng của sản phẩm lên men tự nhiên từ hoa quả với độ rượu nhẹ, bọt ga đầy trắng mịn. Vang Nổ thăng long tạo cảm giác hương phấn, êm dịu, vui tươi, sản phẩm được đóng chai dung tích 750ml...</p><a class="blog__link" href="#">Read more</a>
+        </div>
+      </div>
+>>>>>>> css form login-register-order-contact--responsite
```
Comment (chienpv-0644) [inline dist/pages/blog-page.html]: (3tat)

File: dist/register-page.html [hunk 4]
```diff
@@ -200,7 +203,12 @@ <h1 class="footer__title">Gửi email</h1>
                 <p class="footer__description">Gửi email cho chúng tôi để được hộ trở</p>
                 <div class="footer__input">
                   <div class="footer__email">
+<<<<<<< HEAD
                     <input class="footer__input-email" type="email" placeholder="Enter your email"><a class="footer__link" href="#">Gửi</a>
+=======
+                    <input class="footer__input-email" type="email" placeholder="Enter your email">
+                    <div class="email-center"><a class="footer__link" href="#">Gửi</a></div>
+>>>>>>> css form login-register-order-contact--responsite
```
Comment (chienpv-0644) [inline dist/register-page.html]: (4tat)

**Comment by chienpv-0644** [comment]
commented

**Comment by comadaihiep92** [comment]
ready

**Pull request merged**